soccer/planning/planner/SettlePlanner.cpp

```diff
@@ -282,10 +281,18 @@ Trajectory SettlePlanner::intercept(
                 path.duration() + RJ::Seconds(*_interceptBufferTime) <=
                     ballTime ||
             !fieldRect.containsPoint(ballVelIntercept)) {
+            ball_intercept_maybe = ballVelIntercept;
             break;
         }
     }
 
+    Geometry2d::Point ballVelIntercept;
```

Comment:
Can you throw in a comment about the edge case this will be triggered? I assume it's to deal with the case the ball stops before we would reach it